src/index.js

```diff
@@ -20,3 +21,20 @@ export {default as UsageJsonFormatter} from './formatter/usage_json_formatter'
 export const defineSupportCode = SupportCodeFns.add
 export const getSupportCodeFns = SupportCodeFns.get
 export const clearSupportCodeFns = SupportCodeFns.reset
+
+const cwd = process.cwd()
+const step = helpers.defineStepFactory(cwd)
+const hook = helpers.defineHookFactory(cwd)
+export const defineStep = (pattern, options, code) => {
+  SupportCodeFns.addStepDefinition(step(pattern, options, code))
+}
```

Comment:
Ok, i'll start work on a new branch and close this PR.

Do you really see the need for `defineSupportCode` once we have the other solution?